modules/age.nix

```diff
@@ -103,9 +116,28 @@ in
       message = "age.sshKeyPaths must be set.";
     }];
 
+    # Create a new directory full of secrets for symlinking (this helps
+    # ensure removed secrets are actually removed, or at least become
+    # invalid symlinks).
+    system.activationScripts.agenixMountSecrets = ''
+      _count="$(basename "$(readlink /run/secrets)" || echo 0)"
+      (( ++_count ))
+      echo "[agenix] symlinking new secrets generation $_count to /run/secrets..."
+      mkdir -pm 0750 "${cfg.secretsMountPoint}"
+      mount | grep "${cfg.secretsMountPoint} type ramfs" -q || mount -t ramfs none "${cfg.secretsMountPoint}" -o nodev,nosuid,mode=0750
+      mkdir -pm 0750 "${cfg.secretsMountPoint}/$_count"
```

Comment:
This line triggers [SC2174](https://github.com/koalaman/shellcheck/wiki/SC2174).

This also applies to the `mkdir -pm` above, I suppose.

Reply:
I personally don't see a problem with that (we want the mountpoint directory and the count directory to be 0750; the parents don't really matter that much), but I suppose a `chmod` would be more explicit.